tests/samplers_tests/test_cmaes.py

```diff
@@ -61,6 +61,33 @@ def test_init_cmaes_opts(use_separable_cma: bool, cma_class_str: str) -> None:
         assert actual_kwargs["population_size"] is None
 
 
+@patch("optuna.samplers._cmaes.get_warm_start_mgd")
+def test_warm_starting_cmaes(mock_func_ws: MagicMock) -> None:
+    def objective(trial: optuna.Trial) -> float:
+        x = trial.suggest_uniform("x", -10, 10)
+        y = trial.suggest_uniform("y", -10, 10)
+        return x ** 2 + y
+
+    source_study = optuna.create_study()
+    source_study.optimize(objective, 20)
+    source_trials = source_study.get_trials(deepcopy=False)
+
+    mock_func_ws.return_value = (
+        np.zeros(2),
+        0.0,
+        np.zeros(
+            (
+                2,
+                2,
+            )
+        ),
+    )
```

Comment:
Do you have specific reasons to split lines? If we remove the trailing commas, we can make these a single line.
```suggestion
    mock_func_ws.return_value = (np.zeros(2), 0.0, np.zeros((2, 2)))
```